jackson/src/main/scala/com/lightbend/lagom/internal/jackson/OldJacksonJsonSerializer.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.internal.jackson
+
+import akka.actor.ExtendedActorSystem
+import akka.event.Logging
+import akka.serialization.BaseSerializer
+import akka.serialization.SerializationExtension
+import akka.serialization.SerializerWithStringManifest
+import akka.serialization.jackson.JacksonJsonSerializer
+import akka.serialization.jackson.JacksonMigration
+
+/**
+ * Placeholder for Lagom 1.5.x JacksonSerializer.
+ * Needed for deserialization of messages during rolling update, and old data (events, snapshots)
+ * that were serialized with the Lagom 1.5.x JacksonSerializer.
+ */
+private[lagom] class OldJacksonJsonSerializer(val system: ExtendedActorSystem)
+    extends SerializerWithStringManifest
+    with BaseSerializer {
+
+  private val log = Logging.getLogger(system, getClass)
+  private lazy val jacksonJsonSerializer: SerializerWithStringManifest = {
+    SerializationExtension(system)
+      .serializerByIdentity(JacksonJsonSerializer.Identifier)
+      .asInstanceOf[SerializerWithStringManifest]
+  }
+
+  if (system.settings.config.hasPath("lagom.serialization.json.migrations")) {
+    throw new IllegalStateException(
+      "JacksonJsonSerializer migrations defined in " +
+        s"'lagom.serialization.json.migrations' must be rewritten as [${classOf[JacksonMigration].getName}] " +
+        "and defined in config 'akka.serialization.jackson.migrations'."
+    )
```

Comment:
I would prefer to copy the values from `lagom.serialization.json.migrations` to `akka.serialization.jackson.migrations` and log a warning if we detect that `lagom.serialization.json.migrations` is not empty. 

Maybe not needed to throw an exception.

Reply:
@renatocaval the code reading and using the migrations configured in `lagom.serialization.json.migrations ` is gone so this must throw an exception.

Reply:
see comment https://github.com/lagom/lagom/pull/1955#discussion_r294635553